packages/flutter/lib/src/widgets/text.dart

```diff
@@ -121,6 +126,10 @@ class DefaultTextStyle extends InheritedWidget {
   /// [Text.maxLines].
   final int maxLines;
 
+  /// The strategy to use when calculating the width of the Text. See
```

Comment:
nit: same thing as above here, but additionally, put only one sentence in the first paragraph.

see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements